plugin/src/main/scala/io.github.jonas.paradox.material.theme/ParadoxMaterialThemePlugin.scala

```diff
@@ -26,7 +26,7 @@ object ParadoxMaterialThemePlugin extends AutoPlugin {
     paradoxMaterialTheme / version :=
       Option(ParadoxPlugin.readProperty("paradox-material-theme.properties", "version"))
         .getOrElse(sys.error("Undefined paradox-material-theme version")),
-    paradoxTheme := Some("io.github.jonas" % "paradox-material-theme" % (paradoxMaterialTheme / version).value)
+    paradoxTheme := Some("com.github.sbt" % "sbt-paradox-material-theme" % (paradoxMaterialTheme / version).value)
```

Comment:
I am not 100% sure, because I did not check, but I think the sbt- prefix you added is incorrect here, because the project also publishes a library with such a name (as opposed to the sbt-plugin it publishes)

Reply:
or maybe I mix up things here, if so sorry for the noise...

Reply:
The project is legitimately called sbt-paradox-material-theme now and if its not then we probably should call it that so its consistent with the rest of the community plugins (which all have the sbt prefix)